contracts/delegate.md

```diff
@@ -204,8 +266,12 @@ function provideOrder(
 
 ---
 
-| Revert Reason         | Scenario                                                       |
-| :-------------------- | :------------------------------------------------------------- |
-| `TOKEN_PAIR_INACTIVE` | There is no rule set for this token pair.                      |
-| `AMOUNT_EXCEEDS_MAX`  | The amount of the trade would exceed the maximum for the rule. |
-| `PRICE_INCORRECT`     | The order is priced incorrectly for the rule.                  |
+| Revert Reason              | Scenario                                                       |
+| :------------------------  | :------------------------------------------------------------- |
+| `SIGNER_MUST_BE_SENDER`    | Ensure msg.sender is set as the order signer.                  |
```

Comment:
Picky but... the first 4 `scenarios` start with `Ensure...` which is not a scenario. The last 3 `scenarios` are then actual scenarios.
e.g.
`Ensure sender wallet is set to the tradeWallet. `
->
`The sender wallet is not equal to the trade wallet. `